ci/infra/testrunner/tests/test_skuba_upgrade.py

```diff
@@ -116,7 +124,9 @@ def test_upgrade_apply_from_previous(setup, platform, skuba):
     outs = {}
     for (r, n) in [("master", 0), ("worker", 0)]:
         node = "my-{}-{}".format(r, n)
+        assert node_is_ready(kubectl, node)
         outs[node] = skuba.node_upgrade("apply", r, n)
+        assert node_is_upgraded(kubectl, node)
```

Comment:
In the normal update case we want to check that the update timer is still running, right? (sorry if this is already in another test, I didn't see it)
`assert platform.ssh_run(r, n, "sudo systemctl is-enabled skuba-update.timer || :").find("disabled") == -1`

Reply:
skuba-update.timer being enabled is the default, if everything goes fine we safely can assume that it is enabled (if not the enable would have failed during the upgrade)

we have `test_upgrade_apply_user_lock` that disables skuba-update (and kured) and only then we explicitly check that it stays disabled

so all in all we don't need to check this specific scenario, as we would also have to check if kubelet and cri-o is running e.g. among all the other things we do during the upgrade

Reply:
My thinking is that since there is logic to disable timers, it could happen that we disable something automatically. :) In the event of a refactor or skuba-update pkg update, if we have a test that checks for the timer errors will get caught right away. 

> [...] we would also have to check if kubelet and cri-o is running e.g. among all the other things we do during the upgrade 

Now that you mention it, that's a pretty good idea, too!